From ef42572c5a98cb7b0d9604d2a3fad0f5018444d1 Mon Sep 17 00:00:00 2001
From: siddhesh <siddhesh>
Date: Sun, 22 Jul 2012 16:52:39 +0000
Subject: [PATCH] 	* sh-tdep.c (sh_treat_as_flt_p): Remove unused variable LEN.

---
 gdb/ChangeLog |    4 ++++
 gdb/sh-tdep.c |    2 --
 2 files changed, 4 insertions(+), 2 deletions(-)

2012-07-22  Siddhesh Poyarekar  <siddhesh@redhat.com>

	* sh-tdep.c (sh_treat_as_flt_p): Remove unused variable LEN.

Index: gdb-7.2/gdb/sh-tdep.c
===================================================================
--- gdb-7.2.orig/gdb/sh-tdep.c
+++ gdb-7.2/gdb/sh-tdep.c
@@ -1032,8 +1032,6 @@ sh_next_flt_argreg (struct gdbarch *gdba
 static int
 sh_treat_as_flt_p (struct type *type)
 {
-  int len = TYPE_LENGTH (type);
-
   /* Ordinary float types are obviously treated as float.  */
   if (TYPE_CODE (type) == TYPE_CODE_FLT)
     return 1;
